uniswap/util.py

```diff
@@ -18,7 +18,7 @@
 
 def _get_eth_simple_cache_middleware() -> Middleware:
     return construct_simple_cache_middleware(
-        cache_class=cast(Type[Dict[Any, Any]], functools.partial(lru.LRU, 256)),
+        cache=functools.partial(lru.LRU, 256),  # type: ignore
```

Comment:
Is this correct? I seem to get a lot of chainId hits on my local ganache node...